velox/exec/fuzzer/WindowFuzzer.cpp

```diff
@@ -244,17 +244,36 @@ void WindowFuzzer::go() {
           generateSortingKeysAndOrders("s", argNames, argTypes);
     }
     const auto partitionKeys = generateSortingKeys("p", argNames, argTypes);
-    const auto [frameClause, isRowsFrame] = generateFrameClause();
     const auto input =
         generateInputDataWithRowNumber(argNames, argTypes, signature);
```

Comment:
Shouldn't we add columns for the k-range frame boundary values in `generateInputDataWithRowNumber` ?